src/renderers/testing/ReactTestReconcileTransaction.js

```diff
@@ -57,9 +57,10 @@ var TRANSACTION_WRAPPERS = [ON_DOM_READY_QUEUEING];
  *
  * @class ReactTestReconcileTransaction
  */
-function ReactTestReconcileTransaction() {
+function ReactTestReconcileTransaction(mockConfig) {
+  this._mockConfig = mockConfig;
   this.reinitializeTransaction();
-  this.reactMountReady = CallbackQueue.getPooled(null);
+  this.reactMountReady = CallbackQueue.getPooled(this);
```

Comment:
Since `this.reactMountReady` is accessed via `getReactMountReady()`, let’s add `getMockConfig()` for consistency and use that instead outside.